tools/linux_build.py

```diff
@@ -31,7 +31,7 @@ def stage2(map):
         txt = "-M{}".format(map)
     else:
         txt = ''
-    args = "bash dm.sh {} tgstation.dme".format(txt)
+    args = "bash tools/travis/dm.sh {} tgstation.dme".format(txt)
```

Comment:
I need to change my buildscript so it works again. So every PR I make, I include the change until one of them are merged. So this is already here.